main.cpp

```diff
@@ -0,0 +1,155 @@
+#include <iostream>
+#include <fstream>
+
+
+
+using std::cin;
+using std::cout;
+using std::endl;
+using std::string;
+
+const string CONFIG_FILE_PATH = "/users/outla/Documents/WCCC/Semester 4/CPT-180/Assignment2/app.cfg";
+const string BLANK = "";
+
+
+void init() {
+
+    string name;
+    if(name == BLANK){
+        cout << "Please Enter Your Name (First and Last)" << endl;
+        getline(cin, name);
+    }else{
+        cout << "You Must Enter a Name" << endl;
+    }
```

Comment:
This construction does not persistently ask the user for the value if they enter a blank one